browser/ui/views/commands/default_accelerators_mac.mm

```diff
@@ -62,7 +69,7 @@ AcceleratorMapping ToAcceleratorMapping(NSMenuItem* item) {
                                      characters:keyEquivalent
                     charactersIgnoringModifiers:keyEquivalent
                                       isARepeat:NO
-                                        keyCode:0];
+                                        keyCode:keyCode];
```

Comment:
Passing 0 to this caused the converting routine to fall back to 'A' - https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:ui/events/keycodes/keyboard_code_conversion_mac.mm;l=653;drc=34b556527508d598afa5fe91626d491904aeaaf2